mk/Common.mk

```diff
@@ -76,22 +76,26 @@ $(error Must define a COMPLIANCE_REPO to use the common makefile)
 endif
```

Comment:
This diff of `Common.mk` is a bit difficult to read because `diff` gets confused by the similarity of the lines.
It is easier read by opening the entire file.